presto-product-tests/src/main/java/com/facebook/presto/tests/SystemConnectorTests.java

```diff
@@ -55,10 +57,18 @@ public void selectRuntimeQueries()
                 "  last_heartbeat," +
                 "  'end' " +
                 "FROM system.runtime.queries";
-        assertThat(query(sql))
-                .hasColumns(VARCHAR, VARCHAR, VARCHAR, VARCHAR, VARCHAR, ARRAY,
-                        BIGINT, BIGINT, BIGINT, TIMESTAMP, TIMESTAMP, TIMESTAMP, VARCHAR)
-                .hasAnyRows();
+        if (usingTeradataJdbcDriver(defaultQueryExecutor().getConnection())) {
+            assertThat(query(sql))
+                    .hasColumns(VARCHAR, VARCHAR, VARCHAR, VARCHAR, VARCHAR, VARCHAR,
```

Comment:
put each arg in separate line

Reply:
that would make the change less readable

Reply:
I think that would be too verbose